pkg/skaffold/deploy/deploy.go

```diff
@@ -21,11 +21,14 @@ import (
 	"io"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/graph"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/log"
 )
 
 // Deployer is the Deploy API of skaffold and responsible for deploying
 // the build results to a Kubernetes cluster
 type Deployer interface {
+	log.Logger
```

Comment:
Instead of `log.Logger` being embedded, could `Deployer` just have a `Logger` method?
```
Logger() log.Logger
```

I think this is better than embedding since the logger interface methods can be simplified to just `Start`, `Stop`, `Mute`, etc. without the `...Logger` suffix.